main.tex

```diff
@@ -883,15 +883,20 @@ \subsection{Modeling}
 % Nadia, Lim
 % Tom R. -- unit support
 
-The \package{astropy.modeling} subpackage provides a framework for representing
-analytical models and performing model evaluation and parameter fitting. Models and
-fitters are independent of each other: a model can be fit with different
+Tha \package{astropy.modeling} subpackage provides a framework for representing
+analytical models and performing model evaluation and parameter fitting.
+The main motivation for it was creating a framework
+which allows arbitrary combination of models to support the Generalized World
+Coordinate System (GWCS) package.\footnote{\url{https://github.com/spacetelescope/gwcs}}
+The current FITS WCS model lacks the flexibility to represent
+arbitrary distortions and does not meet the needs of many types of current instrumentation.
+The addition of units to modeling made it attractive for representing
```

Comment:
I don't like the passive voice here. Maybe: "The fact that the \astropypkg modeling framework now supports propagating units also makes it a useful tool for representing and fitting astrophysical models within data analysis tools."